### ISSUE TITLE
Use @all for tagging all chat members

### ISSUE THREAD

**Issue by 3LITA** [pool-party/pull-party-bot#49]
## Scenario:
I want **ppbot** to tag all chat members when `@all` is sent

## Suggested solution:
1. Explicitly show members that will be tagged when `@all` is sent (by using `/all` command)
2. Provide an ability to **add users** to the `@all` party
3. Automatically add users to `@all` when they _join the chat_ or when they _send message_ (use both)
4. Add recommendation in the bot description text to _add it to the just created chat_ (to minimize the chances of missing some users that change their nicknames and stay passive)

**Comment by nothingelsematters** [comment]
> [TODO]: consider using pins for this case

nah, awful idea. plz w8 4 another year for another random comment. stay tuned 🙌
